src/project/internal/projectautosaver.cpp

```diff
@@ -162,10 +162,12 @@ void ProjectAutoSaver::onTrySave()
         return;
     }
 
+#ifndef NDEBUG
```

Comment:
This is not ideal. Maybe let's get rid of it; if we decide later that it is still undesired that autosave runs a corruption check, then we can figure out a nicer way to do it.

Reply:
removed